opentok/opentok-meet
# [OPENTOK-47645] Add fullHD publisher option

#### What is this PR doing?
Add fullHD publisher option

#### How should this be manually tested?
With a 1080 webcam, run meet app and join a session.
- Click on Unpublish, and Publish 1080: check resolution is 1920x1080
For regression testing:
- Click on Unpublish, and Publish 720: check resolution is 1280x720
- Click on Unpublish, and Publish SD: check resolution is 640x480

#### What are the relevant tickets?
OPENTOK-47645
